Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void Demo ()
+        {
+            string userFirstName = GetStringInput(message: "Enter your first name: ");
+            string userSurname = GetStringInput(message: "Enter your surname: ");
+            int userAge = GetIntInput(message: "Enter your age: ");
+            float userHeight = GetFloatInput(message: "Enter your height in cm: ");
+            float userWeight = GetFloatInput(message: "Enter your weight in kg: ");
+            float userBMI = CalculateBmi(userWeight, userHeight);
+            Console.WriteLine(userFirstName + " " + userSurname + " is " + userAge + " years old, his weight is " + userWeight + " kg and his height is " + userHeight + " cm.");
+            Console.WriteLine("BMI = " + userBMI);
+        }
+
+        public static float CalculateBmi(float weight, float height)
+        {
+            float bmi = weight / (height * height);
+            return bmi;
+        }
+
+        public static float GetFloatInput(string message)
+        {
+            Console.Write(message);
+            var floatInput = Console.ReadLine();
+            return float.Parse(floatInput, CultureInfo.InvariantCulture.NumberFormat);
+        }
+
+        public static string GetStringInput(string message)
+        {
+            Console.Write(message);
+            var name = Console.ReadLine();
+            return name;
+        }
+
+        public static int GetIntInput(string message)
+        {
+            Console.Write(message);
+            var age = Console.ReadLine();
+            //return Convert.ToInt32(age);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4863638)